### ISSUE TITLE
Variables to change when forking site

### ISSUE THREAD

**Issue by ubik23** [ubik23/charactercreator#149]
Here is the list of things that should be customized when forking the Character Creator:

(line number, description)
6, description
9, title
11, link
12, og title
13, og description
14, og image
15, url
18-33, Google Analytics script
401, github link
402, facebook link
403, patreon link
404, twitter link
504-508, infolinks code (advertiser), goes with ads.txt
638, overlay title (about)
644-647, overlay content (about)
648, overlay copyright (about)
663-671, F.A.Q
686-688, download message
692, patreon button
693-696, Brave affiliate button
698, invitation to email
699, email address
700, BTC donations message

**Comment by millette** [comment]
Use php on dev side to build.

**Comment by millette** [comment]
After #152

**Comment by ubik23** [comment]
#152 has been merged and unused icons have been removed making the page much lighter! The line numbers are no longer relevant but the description should be enough to figure out what needs to be isolated. Some text elements like the title, might be replaced by an image (SVG) in some cases, like it is the case for our current whitelabel. I don't know if this changes anything but I figured it was worth mentioning. I haven't started to change the branding yet, so this is still relevant at the time of writing.

**Comment by ubik23** [comment]
I can think of a few side benefits to adding php on the dev side of things. The modals (a bunch of them, after the footer) could be organized in separate files and added to the index using php includes. We could do the same with our SVG symbols which make a mess of our index currently. A few includes would make the file much cleaner and easier to work with IMHO.

**Comment by millette** [comment]
php was a quick answer for a single use case, but since we're in a JS world, and using php for the build would force many devs to use php even if it's not in their stack,it might be preferable to use a js solution. I mentionned React, half jokingly, but the truth is it can be used server-side (or at build time, as we would with php). Or use another templating method (straight up jsx, lodash templates, etc.)

**Comment by ubik23** [comment]
Done and merged.
